src/features/player/movement.js

```diff
@@ -93,21 +88,17 @@ export default function handleMovement(player) {
     const tiles = store.getState().map.tiles;
     const y = location[1] / SPRITE_SIZE;
     const x = location[0] / SPRITE_SIZE;
-    console.log(x,y)
-    console.log(tiles[y][x])
     if (tiles[y][x].value === "E") {
-      console.log("end tile")
       removeTileData()
       loadLevel(2)
     }
   }
 
-  function checkPothole(oldPos, newPos) {
+  function checkPothole(oldPos, newPos) { // why oldPos??
       const tiles = store.getState().map.tiles;
       const y = newPos[1] / SPRITE_SIZE;
       const x = newPos[0] / SPRITE_SIZE;
       const nextTile = tiles[y][x];
-      console.log(nextTile.value, "next tile")
       if (nextTile.value === "PB") {
         return true
       }
```

Comment:
should oldPos be used in this function? didn't want to touch it as I didn't look too much into how it worked